app/views/govuk_publishing_components/components/_share_links.html.erb

```diff
@@ -24,6 +24,16 @@
 
     <ul class="gem-c-share-links__list">
       <% links.each do |link| %>
+        <% link_text = capture do %>
+          <span class="govuk-visually-hidden">
+            <% if link[:hidden_text] %>
+              <%= link[:hidden_text] %>
+            <% else %>
+              Share on
```

Comment:
Having made my previous comment, I'm wondering should we default to having hidden text? What's the benefit? (we're making more work for ourselves, once this gets rolled out we'll need to modify any apps that have a workaround or include 'Share on' in their link text already, which we otherwise could do at our leisure).

Reply:
the benefit is that we're actually improve what's being read out. quick look at the usage of the component that in most cases the link text is just the service (Twitter, Facebook).